src/main.js

```diff
@@ -59,15 +74,35 @@ const parseVarFromType = envSpecAsArray => {
 const outputHTML = envValues => {
   //create HTML format
   if (envValues) {
-    envValues = envValues.map(
-      element =>
-        `<label for="env_spec_${element[0].toLowerCase()}">${
-          element[0]
-        }</label>\n` +
-        `<input id="env_spec_${element[0].toLowerCase()}" name="${element[0].toLowerCase()}" type="${
-          element[1]
-        }" />\n`
-    );
+    envValues = envValues.map(element => {
+      switch (element[2]) {
```

Comment:
By implementing the structure object suggested above, this can be simplified to the following:

```javascript
if (entry.type) {
    // Do things for typed entries
} else {
    // Do things for restricted choices
}
```